src/main/scala/com/cognite/sdk/scala/v1/resources/events.scala

```diff
@@ -72,7 +72,12 @@ class Events[F[_]](val requestSession: RequestSession[F])
   override def deleteByIds(ids: Seq[Long]): F[Unit] = deleteByIds(ids, false)
 
   override def deleteByIds(ids: Seq[Long], ignoreUnknownIds: Boolean = false): F[Unit] =
-    DeleteByIds.deleteByIdsWithIgnoreUnknownIds(requestSession, baseUrl, ids, ignoreUnknownIds)
+    DeleteByIds.deleteByCogniteIdsWithIgnoreUnknownIds(
+      requestSession,
+      baseUrl,
+      ids.map(CogniteInternalId.apply),
+      ignoreUnknownIds
+    )
```

Comment:
```suggestion
    deleteByCogniteIds(ids.map(CogniteInternalId.apply), ignoreUnknownIds)
```

Reply:
Maybe add a trait DeleteByCogniteId which will automatically implement these methods

Reply:
Not sure I get what you mean here, I added a trait `DeleteByCogniteIds` and the object for the implementation of method deleteWithIgnoreUnknownIds. The method in trait can be used as a common call in SdkV1Relation.scala of Spark Data Source, but here it uses the method defined in object `DeleteByCogniteIds`, since we need to pass the requestUrl and baseUrl